controlpanel/api/aws.py

```diff
@@ -96,14 +96,14 @@ def iam_assume_role_principal():
     "Version": "2012-10-17",
     "Statement": [
         {
-            "Sid": "console",
```

Comment:
I'd be wary of changing the Sid of the statements. I can't remember if this specific one is used programmatically but at least some of them are (not just in CP but potentially from other tools, e.g. I think `s3tools` was checking the IAM policy to determine permissions at some point?)

Basically I'd leave this to `console` unless there are very good reason not to.

Reply:
:+1:

Reply:
Without going too far, here an example of where some code relies on some of the `Sid` to have certain values (the new migration for example expects this to be `console`):

https://github.com/ministryofjustice/analytics-platform-ops/pull/372/files#diff-1bedc03903ba2fba183c28cf1ea8c8796f8c835f7fa09fbca156d308bff20946R140

Reply:
Great point, but the code you point to is uses the value `console` simply to migrate away from having a SID called `console`. :slightly_smiling_face: